assets/views/home/modEdit.ejs

```diff
@@ -86,6 +86,12 @@
                   ng-if="!refUser.banned">Ban User</button>
           <button type="button" class="btn btn-danger" ng-click="setLocalBan(refUser.name, false)"
                   ng-if="refUser.banned">Unban User</button>
+          <button type="button" class="btn btn-danger has-spinner" ng-click="modClearSessions()"
```

Comment:
Is this supposed to be `modClearSessions(refUser.name)`? It seems like it wouldn't work as-is, although I haven't tested it.

```suggestion
          <button type="button" class="btn btn-danger has-spinner" ng-click="modClearSessions(refUser.name)"
```

Reply:
Yes you're correct